src/analysis/classic/druid/balance/CONFIG.tsx

```diff
@@ -0,0 +1,53 @@
+import Expansion from 'game/Expansion';
+import SPECS from 'game/SPECS';
+import type Config from 'parser/Config';
+import CHANGELOG from './CHANGELOG';
+
+const config: Config = {
+  // The people that have contributed to this spec recently. People don't have to sign up to be long-time maintainers to be included in this list. If someone built a large part of the spec or contributed something recently to that spec, they can be added to the contributors list. If someone goes MIA, they may be removed after major changes or during a new expansion.
+  contributors: [],
```

Comment:
make sure you add your name here :)
```suggestion
  contributors: [iTitou],
```